portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -34,9 +34,12 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     comments.add("test");
     comments.add("comments");
 
-    String json = {"[{"};
-    
+    String json = "[";
+    for (String comment : comments)
+      json = json.concat("comment: " + comment + "},");
+    json = json.concat("]");
+
     response.setContentType("text/html;");
```

Comment:
So the content type tells the browser what kind of content we are returning. Now that you are returning json you might need to change this to "application/json" please double check

Reply:
I fixed it.